docs/customization/health.md

```diff
@@ -0,0 +1,66 @@
+# Health Checks Customization
+
+Argo CD leverages [Lua](https://www.lua.org/) script to implement health assessment for several standard Kubernetes types. This allows to quickly customized health assessments
+and provide health checks for Kubernetes custom resources.
+
+There are two ways to include custom health check.
+
+## Define Custom Health Check in `argocd-cm` ConfigMap
+
+Custom Health checks can be defined in `resource.customizations` field of `argocd-cm`. The `resource.customizations` is a YAML map where each key is a resource group kind and
+value is a object with the Lua script in the `health.lua` field. Following example demonstrates a health check implementation for `certmanager.k8s.io/Certificate`.
+
+```yaml
+data:
+  resource.customizations: |
+    certmanager.k8s.io/Certificate:
+      health.lua: |
+        hs = {}
+        if obj.status ~= nil then
+        if obj.status.conditions ~= nil then
```

Comment:
There seems to be a missing level of indentation.

Reply:
fixed